language/stdlib/modules/Genesis.move

```diff
@@ -1,8 +1,10 @@
-// The genesis module. This defines the majority of the Move functions that
-// are executed, and the order in which they are executed in genesis. Note
-// however, that there are certain calls that remain in Rust code in
-// genesis (for now).
 address 0x1 {
+
+/// The `Genesis` module defines the Move initialization entry point of the Libra framework
+/// when executing from a fresh state.
+///
+/// > TODO: Currently there are a few additional functions called from Rust during genesis.
```

Comment:
I tried making genesis a single scripts (no module, no Rust code) the other day and got pretty close but it was too big of a change before the freeze... If I had to start over I think I would insist that genesis had to be a single Move script from the beginning but allow it to be codegen'd from Rust.

Reply:
I would prefer it not to be codegen'd for reasons of transparency and verifiability. I could imagine there is a (possibly complex) data structure for configuring it. As we can't pass those into scripts for now, perhaps the current module approach also has advantages? (besides that it may be not complete)